### ISSUE TITLE
RedisConnectionException: SlaveConnectionPool no available Redis entries

### ISSUE THREAD

**Issue by simpleyetawesomeprojects** [redisson/redisson#5643]
Hello,

I'm testing Redisson with AWS ElasticCache and seeing an issue.

I have a clustered configuration with only a single node (No replications/No slaves).
My configuration is based on the YAML config file. I have copied the exact config from the Redisson [documentation](https://github.com/redisson/redisson/wiki/2.-Configuration/#242-cluster-yaml-config-format) and updated only the 'nodeAddresses' with the ElasticCache endpoint.

All the set operations worked with the API. However, when I added code to read/write a `set` object, I saw the error `RedisConnectionException: SlaveConnectionPool no available Redis entries`

The application did not recover and I had to restart all the pods. However, the issue still occurred after the pod restart.

**Expected behavior**
Per the documentation, Redisson client should connect to Master and work if slaves are unavailable
`readMode: "SLAVE" , Read from slave nodes, uses MASTER if no SLAVES are available,`.  

**Actual behavior**
```
"org.redisson.client.RedisConnectionException","error.message":"SlaveConnectionPool no available Redis entries. Master entry host: 172.31.17.121/172.31.17.121:6379","error.stack_trace":"org.redisson.client.RedisConnectionException: SlaveConnectionPool no available Redis entries. Master entry host: 172.31.17.121/172.31.17.121:6379
	at org.redisson.connection.pool.ConnectionPool.get(ConnectionPool.java:105)
	at org.redisson.connection.pool.SlaveConnectionPool.get(SlaveConnectionPool.java:31)
	at org.redisson.connection.MasterSlaveEntry.connectionReadOp(MasterSlaveEntry.java:629)
	at org.redisson.command.RedisExecutor.connectionReadOp(RedisExecutor.java:792)
	at org.redisson.command.RedisExecutor.getConnection(RedisExecutor.java:708)
	at org.redisson.command.RedisExecutor.execute(RedisExecutor.java:127)
	at org.redisson.command.RedisExecutor$1.run(RedisExecutor.java:323)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
	at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
	at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
	at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
	at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:750)

```
**Steps to reproduce or test case**
Create a clustered ElasticCache with only 1 node (No replicas, No slaves)

**Redis version**
7.1.0

**Redisson version**
3.26.1
`implementation("org.redisson:redisson-spring-boot-starter:3.26.1") {
        exclude group: 'org.redisson', module: 'redisson-spring-data-31'
    }
 implementation 'org.redisson:redisson-spring-data-27:3.26.1'`

**Redisson configuration**
Exact config from the Redisson [documentation](https://github.com/redisson/redisson/wiki/2.-Configuration/#242-cluster-yaml-config-format) and updated only the 'nodeAddresses' with the ElasticCache endpoint.

Any help is appreciated.

PS:
I'm also yet to evaluate the PRO version based on the open-source version capabilities.

**Comment by simpleyetawesomeprojects** [comment]
Update:
changing the below have worked
`  readMode: "MASTER"
  subscriptionMode: "MASTER"
`

**Comment by mrniko** [comment]
Fixed in 3.27.0 version.
